frontend/src/scenes/insights/InsightTooltip/InsightTooltip.scss

```
@@ -0,0 +1,89 @@
+@import '~/vars';
+
+.ph-graph-tooltip {
+    padding: 0 !important;
+    overflow: hidden;
+    border: 1px solid $border;
+    box-shadow: 0 4px 12px rgba(0, 0, 0, 0.1);
+    font-size: 14px;
+    z-index: $z_graph_tooltip;
+    transition: all 0.4s;
+    max-width: 480px;
+    border-radius: $radius;
+    background: #fafafa;
+
+    .LemonTable {
+        font-size: 0.75rem;
+        border: none;
+        border-bottom-left-radius: 0;
+        border-bottom-right-radius: 0;
+        border-bottom: 1px solid $border;
+
+        .color-column {
+            padding: 0;
+            width: 6px;
+        }
+
+        .datum-column {
+            padding-left: 0.5rem;
+        }
+
+        .LemonTable__scroll {
+            width: 100%;
+            overflow: hidden;
+        }
+        .LemonTable__content > table > thead {
+            text-transform: none;
```

In this case I also suggest overriding `letter-spacing`, as it's normally increased in the header to make the all-caps titles more relaxed.
```suggestion
            text-transform: none;
            letter-spacing: normal;
```
BTW: you may consider making this a prop on `LemonTable`, something like `uppercaseHeader` (which would be `true` by default) – I could see the alternative style being useful outside of just the tooltips

---

moved into a prop